dll/win32/iernonce/thread.cpp

```diff
@@ -0,0 +1,19 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Code to execute entries in RunOnceEx
+ * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+#include "thread.h"
+#include "registry.h"
+#include "dialog.h"
+
+unsigned __stdcall RunOnceExExecThread(void *Param)
+{
+    ProgressDlg *pProgressDlg = (ProgressDlg *)Param;
+
+    pProgressDlg->m_RunOnceExInst.Exec(pProgressDlg->m_hWnd);
+
+    return 0;
+}
```

Comment:
I'm not convinced this needs to be in its own separate file. It could be put inside the main iernonce.cpp file.

Reply:
at first, I thought this file would be big but after several times refactor, almost all the code was moved out 🤣
I will remove it soon!